integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -194,25 +194,35 @@ jobMaster:
 # UFS journal with HDFS example
 # journal:
 #   type: "UFS"
+#   ufsType: "non-local"
 #   folder: "hdfs://{$hostname}:{$hostport}/journal"
 # EMBEDDED journal to /journal example
 # journal:
 #   type: "EMBEDDED"
 #   folder: "/journal"
 journal:
-  type: "UFS" # "UFS" or "EMBEDDED"
-  ufsType: "local" # Ignored if type is "EMBEDDED". "local" or "HDFS"
```

Comment:
Semantically `"HDFS"` just meant "non-local". We only use this value as a boolean check for `.Values.journal.ufsType == "local"` in one location, so I think the semantic change is safe to make.

Reply:
Great question! After some thoughts I figure the current `local/hdfs` switch makes more sense.

The current set is [local, hdfs] because these are the only 2 options that we support. Only special setup is needed for "local" in the statefulset.yaml. 

I think by having `local/non-local` vs `local/hdfs`, you change a multiple choice into a binary choice. If we bring in a 3rd option X, then X is either local or non-local. Then you will need yet another property to tell if this is a local disk, or option X. 

By going with `local/hdfs`, you will add one more option `local/hdfs/X`. I think in the short run having fewer config properties is better as long as we don't have too many options like `local/hdfs/A/B/C...Z`.